org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JDTDelegateCommandHandler.java

```diff
@@ -95,6 +98,19 @@ public Object executeCommand(String commandId, List<Object> arguments, IProgress
 					return ProjectCommand.getProjectSettings((String) arguments.get(0), (ArrayList<String>) arguments.get(1));
 				case "java.project.getClasspaths":
 					return ProjectCommand.getClasspaths((String) arguments.get(0), JSONUtility.toModel(arguments.get(1), ClasspathOptions.class));
+				case "java.project.updateSourcePaths": {

```

Comment:
What do you think of renaming it to "java.project.updateClassPaths"? We can support the source path update first, and extend it to other classpath entries on demand.

Reply:
Good idea 👍